instrumentation/opentelemetry-instrumentation-aio-pika/README.rst

```diff
@@ -0,0 +1,22 @@
+OpenTelemetry aio_pika Instrumentation
```

Comment:
```suggestion
OpenTelemetry Aio-Pika Instrumentation
```

Reply:
Done

Reply:
I am not seeing these changes that are marked "Done".